device/options.go

```diff
@@ -27,6 +27,10 @@ const (
 	DefaultDeviceMessageQueueSize = 100
 )
 
+type wrpSourceCheckConfig struct {
```

Comment:
why is this private?

Reply:
We don't need it elsewhere outside the package. We do need the "Type" to be exported for viper to work properly.